main.js

```diff
@@ -1,11 +1,15 @@
+/* global coordinates */
+
+var $carContainer = document.querySelector('.car-container');
 var $car = document.querySelector('img');
 
-document.addEventListener('keydown', turnCar);
-function turnCar(event) {
+document.addEventListener('keydown', moveCar);
+function moveCar(event) {
   turnRight();
   turnLeft();
   turnUp();
   turnDown();
```

Comment:
Instead of calling all these functions in a row, I would check what the `event.key` is here, and call the appropriate function for that key. With the way this is written right now, it looks like you are turning the car in all those directions at once